### ISSUE TITLE
Documenting the Javascript User API

### ISSUE THREAD

**Issue by buggyj** [Arlen22/TW5Docs#2]
There is curently the 'dev' docs where js macros are mentioned, eg http://tiddlywiki.com/dev/index.html#JavaScript%20Macros which I feel is in the wrong place as there is nothing 'dev' about writing macros. There should be a js users guide, separate from a dev guide.

**Comment by Arlen22** [comment]
Are you talking about a place to document "power user" features? Like JavaScript macros, filters, widgets, and such things?

**Comment by buggyj** [comment]
yes, for javascript users, writing js macros may be easier than wikitext macros. Also I feel the dev guide should be a reference for tiddlywiki internals, and guides for writing new filters and widgets  should go into separate documents.

**Comment by Arlen22** [comment]
I believe part of the reason for this was so that everyone didn't just switch back to the way they were used to with TiddlyWiki classic. However, the API and TW5 as a whole has matured to the point that this could probably be beneficial. And especially documenting what parts of $tw are fine to use, and what parts are guts level  and shouldn't be used in a macro.
